package.json

```diff
@@ -18,7 +18,7 @@
     "@types/jest": "^24.0.23",
     "@types/lodash": "^4.14.149",
     "ajv": "^6.10.2",
-    "ava": "~2.4.0",
+    "ava": "^3.13.0",
```

Comment:
Please also update all other `ava` dependencies in each package.  We hoist all dependencies so ideally they should all confirm to the same version.

Reply:
> Out of curiosity: Is there a specific reason to fix exactly this vulnerability?

No there is not. The risk profile is practically nill. I just had some audit warnings crop up that we'rent there the day before, and so I attempted to find the root cause and submit PR as a courtesy. I would not be heartbroken if you decide it's not worth the hassle.

That said, I'll go ahead and make those updates to `ava` dependencies, and anything else you think we need as a result.